src/main/java/com/liferay/faces/util/application/ResourceVerifierFactory.java

```diff
@@ -31,21 +32,22 @@
 public abstract class ResourceVerifierFactory implements FacesWrapper<ResourceVerifierFactory> {
 
 	/**
-	 * @return  an instance of {@link ResourceVerifier} from the {@link ResourceVerifierFactory} found by the {@link
-	 *          FactoryExtensionFinder}.
+	 * Create (if necessary) and return an instance of {@link ResourceVerifier} from the {@link ResourceVerifierFactory}
+	 * found by the {@link FactoryExtensionFinder}.
```

Comment:
Please add the sentence: "The returned instance is designed to be created and used during execution of a request thread."